pkg/util/deployment/deployment.go

```diff
@@ -40,6 +40,12 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// DesiredReplicasAnnotation is the desired replicas for a deployment recorded as an annotation
+	// in its replica sets.
+	DesiredReplicasAnnotation = "deployment.kubernetes.io/desired-replicas"
```

Comment:
Why would we want another controller for scaling multiple replica sets when we already have a controller for replica sets?